odoo/tools/misc.py

```diff
@@ -802,6 +809,44 @@ def deco(*args, **kwargs):
                 return func(*args, **kwargs)
         return deco
 
+    def emit(self, record):
+        pass
+
+
+class quiet_logger(logging.Handler):
```

Comment:
The naming is confusing, this is neither a "quiet logger" nor quite something which quiets a specific (named) logger (like `mute_logger`).